plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,527 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2021, Tyler Gates <tgates81@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: spectrum_model_attrs
+short_description: Enforce a model's attributes in CA Spectrum.
+description:
+    - This module can be used to enforce a model's attributes in CA Spectrum.
+version_added: 2.2.0
```

Comment:
Latest community.general release was [2.4.0](https://github.com/ansible-collections/community.general/issues/582#issuecomment-810150215), this should change to 2.5.0 then.
```suggestion
version_added: 2.5.0
```